packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -30,16 +30,18 @@ export const selectFaceMatchStateBeforeStart = createLivenessSelector(
   (state) => state.context.faceMatchStateBeforeStart
 );
 
+const selectFaceMatchPercentage = createLivenessSelector(
+  (state) => state.context.faceMatchAssociatedParams?.faceMatchPercentage
+);
+
 export interface HintProps {
   hintDisplayText: Required<HintDisplayText>;
 }
 
 const defaultToast = (text: string, isInitial = false) => {
   return (
     <Toast size="large" variation="primary" isInitial={isInitial}>
-      <View aria-live="assertive" aria-label={text}>
-        {text}
-      </View>
+      <View aria-live="assertive">{text}</View>
```

Comment:
removed various aria-labels where there was already an aria-live attribute because osx voice over would double say the aria live and aria label which was extremely annoying